tests/settings.py

```diff
@@ -56,6 +56,18 @@
             }
         }
     }
+elif os.environ.get('IMPORT_EXPORT_TEST_TYPE') == 'postgres':
+    IMPORT_EXPORT_USE_TRANSACTIONS = True
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': 'import_export',
+            'USER': 'postgres',
+            'PASSWORD': 'postgres',
+            'HOST': 'localhost',
```

Comment:
When I run tests, I receive  *django.db.utils.OperationalError: FATAL:  role "postgres" does not exist* error.

Leaving both `USER` and `PASSWORD` empty used my default user to connect to postgresql server.

I am not an postgre expert so it could be that I just have weird local setup?